### DISCUSSION TITLE
dump hangs on vtiger_modtracker_detail

### DISCUSSION THREAD

**Discussion by TonyM15** [YetiForceCompany/YetiForceCRM#17066] [General]
mysqldump: Error 2013: Lost connection to MySQL server during query when dumping table `vtiger_modtracker_detail
Tried many various configs. Table looks OK. Can it be some weird records in it?
By the way vtiger_modtracker_detail dont have unique column. Adding it will cause some unexpected behavior?

**Comment by rskrzypczak** [comment]
It looks like you have exceeded some timeout parameter in your database configuration.
Presumably you have a very large database and if you are still doing a per row dump it may take a while.

> By the way vtiger_modtracker_detail dont have unique column. Adding it will cause some unexpected behavior?

It shouldn't be a problem
